chaoskube/chaoskube.go

```diff
@@ -41,6 +41,8 @@ type Chaoskube struct {
 	DryRun bool
 	// a function to retrieve the current time
 	Now func() time.Time
+	// minimum age of pods to consider
+	MinimumAge time.Duration
```

Comment:
Let's group the fields a little. Can we put this after Timezone and before Logger (you would also change the order of the flags etc.)